src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/EmploymentCheck/RefreshEmploymentChecksCommandHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+using SFA.DAS.EmployerIncentives.Commands.Persistence;
+using SFA.DAS.EmployerIncentives.Commands.Types.ApprenticeshipIncentive;
+
+namespace SFA.DAS.EmployerIncentives.Commands.ApprenticeshipIncentive.EmploymentCheck
+{
+    public class RefreshEmploymentChecksCommandHandler : ICommandHandler<RefreshEmploymentChecksCommand>
+    {
+        private readonly IApprenticeshipIncentiveDomainRepository _incentiveDomainRepository;
+        private readonly ICommandPublisher _commandPublisher;
+
+        public RefreshEmploymentChecksCommandHandler(IApprenticeshipIncentiveDomainRepository incentiveDomainRepository, ICommandPublisher commandPublisher)
+        {
+            _incentiveDomainRepository = incentiveDomainRepository;
+            _commandPublisher = commandPublisher;
+        }
+        public async Task Handle(RefreshEmploymentChecksCommand command, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            var incentivesWithLearning = await _incentiveDomainRepository.FindIncentivesWithLearningFound();
+
+            foreach(var incentive in incentivesWithLearning)
+            {
+                await _commandPublisher.Publish(new SendEmploymentCheckRequestsCommand(incentive.Id));
```

Comment:
I don't think does what's needed as that won't create the employment check records if they don't exist or replace them if they do exist will it?